diagnostics/stc_annular_modes/doc/stc_annular_modes.rst

```diff
@@ -0,0 +1,222 @@
+day of year. It makes four kinds of figures from provided model data:
+
+1. EOF1 pattern plots on standard pressure levels, representing the 
+   annular mode structures throughout the troposphere and stratosphere.
+   (cf., Gerber et al. 2010, Fig. 4; Simpson et al., 2011, Fig. 1)
+2. Annular mode interannual standard deviation (cf., Gerber et al., 2010,
+   Fig. 7)
+3. Annular mode e-folding timescales (or "persistence"; 
+   cf., Gerber et al. 2010, Fig. 8; Kidston et al., 2015, Fig. 1)
+4. Annular mode predictability (cf., Gerber et al., 2010, Fig. 9)
+   
+All figures are made for both hemispheres. This POD also outputs the
+computed annular mode indices and EOF structures as netcdf files. 
+
+The figure from (1) should always be viewed to verify the spatial structure 
+of the annular modes; if the spatial patterns for model data do not match the 
+observations (or the figures in the papers referenced above), then the 
+1st EOF may not represent an "annular mode"-like pattern, which means caution 
+is warranted for interpreting the other figures and the digested output data.
+
+The figures from (2) highlight the seasonal cycle in annular mode variability.
+The figures from (3) show estimates of the seasonally varying persistence of 
+the annular modes. Lastly, the figures from (4) demonstrate what fraction
+of the variance of the annular mode at a given pressure level (default 850 hPa)
+can be "predicted" using a persistence forecast of the annular mode at other 
+levels (see Gerber et al., 2010 for full details).
+
+
+Version & Contact info
+----------------------
+
+- Version/revision information: v1.0 (Mar 2023)
+- Project PIs: Amy H. Butler (NOAA CSL) and Zachary D. Lawrence (CIRES / NOAA PSL)
+- Developer/point of contact: Zachary Lawrence (zachary.lawrence@noaa.gov)
+
+Open source copyright agreement
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The MDTF framework is distributed under the LGPLv3 license (see LICENSE.txt).
+
+
+Functionality
+-------------
+
+This POD is composed of three files, including the main driver script
+``stc_annular_modes.py``, the functions that perform the diagnostic
+computations in ``stc_annular_modes_calc.py``, and the functions that 
+compile the specific POD plots in ``stc_annular_modes_plot.py``. 
+The driver script reads in the necessary data, calls the computation
+functions to perform the anomaly calculations, the EOF analysis, and 
+annular mode diagnostics.
+
+The observational data this POD uses is based on ERA5 reanalysis
+(Hersbach, et al., 2020) zonal mean geopotential heights.
+
+By default, the annular modes are computed for input model data as 
+described in Gerber et al., 2010; in short, for every timestep, the 
+global mean geopotential height is removed, and then a 60 day low-pass 
+filter is applied across the daily data before a 30 year low-pass filter 
+is applied across the days of year. This process is intended to remove 
```

Comment:
Perhaps this is explained in Gerber et al. 2010, but its not intuitive to me why a double filter is needed.

Reply:
The two separate filters (one across time, and one across days of year) are to compute a slowly varying climatology that may have decadally-varying trends. This is illustrated in the Gerber et al. 2010 paper in their Figure 2b (posted below). The jagged/thicker line shows anomalies from removing the global mean heights and the standard seasonal cycle; the thin smooth line shows the slowly varying climatology that is produced from the two filters, which is used to obtain the anomalies in panel c. 

![image](https://user-images.githubusercontent.com/4175294/228560682-46d0f786-75a6-422f-9320-5d5396d948b3.png)